scripts/runClangFormat.sh

```diff
@@ -1,16 +1,12 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```
Otherwise the script will fail on systems such as NixOS that use non-standard paths for their binaries

Reply:
That's correct, but `/bin/sh` is standardised and systems like NixOS do ensure that it is always present.
You argue against `/bin/sh` but suggest to use `/usr/bin/env`, which is the same story.
NixOS does ensure that `/usr/bin/env` always exists, but in the end it is a handled just like `/bin/sh`:
```
[root@nixos:~]# ls -l /bin/sh /usr/bin/env
lrwxrwxrwx 1 root root 75 Jan 23 18:09 /bin/sh -> /nix/store/3xp6zj3gj0qwgz7nq2wcslwvak493srs-bash-interactive-4.4-p23/bin/sh
lrwxrwxrwx 1 root root 66 Jan 23 18:09 /usr/bin/env -> /nix/store/w32xfbwc1yix3q0f3xz4fvhz89cwr73f-coreutils-8.32/bin/env
```

Reply:
I have once been told by some NixOS users that scripts should always use `!/usr/bin/env xy` in order to only require this specific binary at that specific location. That's what I tried to stick with since. Admittedly, I have never actually used or checked out NixOS myself :shrug:

Reply:
Yeah, tl;dr: `/bin/sh` is equally fine and guaranteed to do the right thing on NixOS.